examples/bun-rl/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "bun-rl",
+  "module": "index.ts",
+  "type": "module",
+  "workspaces": [
+    "../../*"
+  ],
```

Comment:
Since bun doesn't support `file:` specifiers, I used workspaces to resolve our monorepo packages.